### PR TITLE
[cairomm] Remove unnecessary dependencies

### PR DESCRIPTION
**Pull request by xiaozhuai**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Review by Adela0814 — DISMISSED** [review]

File: ports/cairomm/usage [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cairomm provides usage:
 
     find_package(PkgConfig REQUIRED)
-    pkg_check_modules(CAIROMM REQUIRED IMPORTED_TARGET cairomm-1.16)
-    target_link_libraries(main PRIVATE PkgConfig::CAIROMM)
+    pkg_check_modules(cairomm REQUIRED IMPORTED_TARGET cairomm-1.16)
+    target_link_libraries(main PRIVATE PkgConfig::cairomm)
```
Comment (dg0yt) [inline ports/cairomm/usage:5]: IMO this shouldn't be changed.
- The change is not necessary.
- AFAICT common practice is `pkg_check_modules(<UPPER_CASE_PREFIX> ...)`.
- The upper-case prefix makes is more consistent with the function call's output variables. (Not used here.)
- The upper-case prefix nicely contrasts with the module name (here: `cairomm-1.16`).

I acknowledge that it is more ugly when used as part of the output target (here: `PkgConfig::CAIROMM`).

Comment (xiaozhuai) [reply ports/cairomm/usage:5]: > I acknowledge that it is more ugly when used as part of the output target (here: `PkgConfig::CAIROMM`).

Yes, that's why I change it. : )
And I also check `cairo`'s usage

```
The package cairo can be imported via CMake FindPkgConfig module:

    # cairo
    find_package(PkgConfig REQUIRED)
    pkg_check_modules(cairo REQUIRED IMPORTED_TARGET cairo)
    target_link_libraries(main PkgConfig::cairo)
```

For sure, the change is not necessary. It's just a preference.

Comment (dan-shaw) [reply ports/cairomm/usage:5]: I agree with @dg0yt. Was there an issue you were trying to fix?

Comment (xiaozhuai) [reply ports/cairomm/usage]: Ok, I've revert this change.

**Review by Adela0814 — APPROVED** [review]

File: ports/cairomm/vcpkg.json [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
   "homepage": "https://www.cairographics.org",
   "license": "LGPL-2.0-only",
   "supports": "!xbox",
   "dependencies": [
     "cairo",
     "libsigcpp",
-    {
-      "name": "pkgconf",
-      "host": true
-    },
```
Comment (dan-shaw) [inline ports/cairomm/vcpkg.json:14]: Why do we remove `pkgconf` as a dependency?

Comment (xiaozhuai) [reply ports/cairomm/vcpkg.json:14]: It's only required on usage time. It doesn't mean it depends on pkgconf.

**Pull request merged**